### ISSUE TITLE
CursorTypeAndBooleanConverter is not working

### ISSUE THREAD

**Issue by soi013** [runceel/Livet#49]
CursorTypeAndBooleanConverter is not working. 
Because, it should convert to **string** instead of CursorType.

```csharp
private CursorType _convertWhenTrue;
        public CursorType ConvertWhenTrue
        {
            get { return _convertWhenTrue; }
            set
            {
                _convertWhenTrue = value;
                _isConvertWhenTrueSet = true;
            }
        }
```

👇

```csharp
private string _convertWhenTrue;
        public string ConvertWhenTrue
        {
            get { return _convertWhenTrue; }
            set
            {
                _convertWhenTrue = value;
                _isConvertWhenTrueSet = true;
            }
        }
```

https://www.c-sharpcorner.com/forums/how-to-bind-wait-cursor-using-mvvm-in-wpf

I can write fix code and PR by hand.
But, this class is generated.
Sorry, I don't know T4 template.

**Comment by runceel** [comment]
Thank you for reporting it.
I didn't know it.

Could you create the PR?
After I merge it, I'll fix T4 template.

**Comment by soi013** [comment]
I wrote PR.

**Comment by runceel** [comment]
Thank you!

**Comment by runceel** [comment]
I have released v3.2.3. It includes your PR.
